codemagic.yaml

```diff
@@ -81,14 +85,24 @@ workflows:
       events:
         - push
       branch_patterns:
-        - pattern: main
+        - pattern: 'main'
+    environment:
+      vars:
+        # Increase the clone depth to 50 to cover all commits from the latest tag
+        CM_CLONE_DEPTH: 50
+      groups:
+        - firebase_credentials
```

Comment:
We need to add the environment groups here again. It seems to add the `environments > vars` above overrides the environment that we import from [configure_environment](https://github.com/nimblehq/android-templates/pull/558/files#diff-bf0ae6a913219a09775a2f777446c5ebaa55b75a6f39c47bd39ac8ce45b2693aR83)